R/backend-postgres.R

```diff
@@ -82,7 +82,34 @@ sql_translate_env.PostgreSQLConnection <- function(con) {
           sql_expr(EXTRACT(QUARTER %FROM% !!x))
         }
       },
-
+      seconds = function(x) {
```

Comment:
Could you please provide a comment that links to the docs for this cast? And can you please move so to the end of the lubridate functions? `quarter` and `wday` should be close together because they're the same sort of function.